docs/source/other/faq.rst

```diff
@@ -1,8 +1,7 @@
 Frequently Asked Questions
 ##############################################################################
 
-The topics application is the main focus of the CS Field Guide website, as it
-contains the majority of educational material for the project.
+The topics application is the main focus of the CS Field Guide website, as it contains the majority of educational material for the project.
```

Comment:
There is no topics application. I assume this should be the chapters application